pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -77,28 +77,29 @@ type kubenetNetworkPlugin struct {
 	hairpinMode     componentconfig.HairpinMode
 	hostPortMap     map[hostport]closeable
 	iptables        utiliptables.Interface
+	cniPath         string
```

Comment:
please add a comment either here or above newplugin about what this field is and what the default is/how it interacts with defaultcniplugindir and which takes precedence.